tests/storage/restore.go

```diff
@@ -562,7 +562,9 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 					Expect(newVMInterfaces[0].MacAddress).ToNot(Equal(oldVMInterfaces[0].MacAddress))
 
 					By("Making sure new VM is runnable")
-					tests.RunVMAndExpectLaunchWithRunStrategy(virtClient, newVM, v1.RunStrategyAlways)
+					err := virtClient.VirtualMachine(vm.Namespace).Start(context.Background(), newVM.Name, &v1.StartOptions{})
```

Comment:
Do you know if this starts the VM, and sets the run strategy to `RunStrategyAlways`?

Reply:
Yes, here is how it is being done:
https://github.com/kubevirt/kubevirt/blob/8016033c66e08163493ebdabd80bceb5d8329800/pkg/virt-api/rest/subresource.go#L526